docs/tutorials/fundamentals/viewer.md

```diff
@@ -342,7 +342,8 @@ The console (when available) appears at the bottom of the viewer as shown below:
 
 #### 2D/3D button  
 
-The second button from the left is the 2D/3D button which toggles between `2D` and `3D` renderings of the data. Run the following code:
+The second button from the left is the 2D/3D button which toggles between `2D` and `3D` renderings of the data. 
```

Comment:
I would add that its tooltip says "Toogle ndisplay", it's not immediately obvious. However now I'm thinking we should probably do this for all buttons - maybe in a follow-up PR?

Reply:
Should we change the heading to `Toggle ndisplay (2d/3d) button`?

Reply:
Done